FluentTc/Locators/MoreOptionsHavingBuilder.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Xml;
+
+namespace FluentTc.Locators
+{
+    public interface IMoreOptionsHavingBuilder
+    {
+        IMoreOptionsHavingBuilder WithCleanSources();
+        IMoreOptionsHavingBuilder RebuildAllDependencies();
+        IMoreOptionsHavingBuilder QueueAtTop();
+        IMoreOptionsHavingBuilder AsPersonal();
+        IMoreOptionsHavingBuilder WithComment(string comment);
+    }
+
+    public class MoreOptionsHavingBuilder : IMoreOptionsHavingBuilder
```

Comment:
MoreOptionsHavingBuilder should not be visible by user of FluentTc. Usually such classes are internal